examples/storybook/.storybook/main.js

```diff
@@ -0,0 +1,13 @@
+module.exports = {
+  "stories": [
```

Comment:
has this file been formatted with prettier?

Reply:
apparently no :) done here 9b229be